chill-scala/src/main/scala/com/twitter/chill/KryoBase.scala

```diff
@@ -48,9 +47,29 @@ class KryoBase(classResolver: ClassResolver, referenceResolver: ReferenceResolve
   protected var strategy: Option[InstantiatorStrategy] = None
 
   val functions: Iterable[Class[_]] =
-    (0 to 22).map { idx =>
-      Class.forName("scala.Function" + idx.toString, true, Thread.currentThread().getContextClassLoader())
-    }
+    List(
```

Comment:
thanks for fixing this.... some of this code is pretty old and really not optimally written but also used in a variety of big systems (many versions of spark, scalding, and a few other systems) so being committed to binary compatibility really helps avoid nasty diamond dependency problems.